src/selectors/entities/apps.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {GlobalState} from 'types/store';
+import {AppBinding} from 'types/apps';
+
+// This file's contents belong to the Apps Framework feature.
+// Apps Framework feature is experimental, and the contents of this file are
+// susceptible to breaking changes without pushing the major version of this package.
+export function getAppBindings(state: GlobalState, location?: string): AppBinding[] {
```

Comment:
If this is being used in mapStateToProps or a similar hook, this will need to be changed to a memoized selector since it otherwise returns new arrays every time its called which will cause unnecessary re-renders.